### DISCUSSION TITLE
Request: 子ページを含めたリンクを共有できるようにする

### DISCUSSION THREAD

**Discussion by kota65535** [growilabs/growi#7733] [Ideas / Requests]
## Informations
現在のShare Linkではそのページのみアクセスすることが可能ですが、子ページ全体を含めてアクセス可能な共有リンクを作成できないでしょうか？

**Comment by yuki-takei** [comment]
@kota65535 ご要望ありがとうございます。今のところそういった機能の検討はしておりません。

需要はありそうですが、ロジックが難しいですね…
ページの階層は、下位に行くほど閲覧権限は厳しくなるので、一律「子ページ全体」へアクセス可能にするわけにはいかないと思います。この件は Discussions 行きな気がしますがどうでしょう。
